.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,98 @@
+name: Bug report
```

Comment:
A lot of incoming bug reports are questions/requests for help.  Like "I have geo JSON in an Oracle blob and it's not working with Superset."  Can we add something - a markdown section at top ? - saying:

Hello Superset Community member!  Please keep things tidy by putting your post in the proper place:
- **Reporting a bug🐛**: use this form.
- **Asking a question or getting help🙏**: post in the [Superset Slack chat](http://bit.ly/join-superset-slack) or [GitHub Discussions](https://github.com/apache/superset/discussions) under "Q&A / Help".
- **Requesting a new feature💡:** Search [GitHub Discussions](https://github.com/apache/superset/discussions) to see if it exists already.  If not, add a new post there under "Ideas".